compiler/repl-service/server/src/main/scala/com/digitalasset/daml/lf/ReplServiceMain.scala

```diff
@@ -217,7 +217,7 @@ class ReplService(
       TimeProvider.UTC)
     runner.runWithClients(clients).onComplete {
       case Failure(e) =>
-        System.err.println(s"$e")
+        println(s"$e")
```

Comment:
The REPL tests only read stdout. But, also to be consistent with other error printing in this module.